roles/postgres/tasks/main.yml

```diff
@@ -61,12 +61,14 @@
   ansible.builtin.template:
     src: postgres-backups@.service.j2
     dest: /usr/lib/systemd/system/postgres-backups@.service
+    mode: '644'
```

Comment:
nit: I find `mode: u=rw,g=r,o=r` is a bit more readable.

Reply:
Done!